### DISCUSSION TITLE
Things we decieded against on

### DISCUSSION THREAD

**Discussion by Ari24-cb24** [aridevelopment-de/datetimeparser#174] [Pinned informations]
This thread will include all things we decieded against on

**Comment by Ari24-cb24** [comment]
<blockquote>quarter two hour</blockquote>
   
 No one uses this grammar in english and it just sounds weird

**Comment by Ari24-cb24** [comment]
The parser does not work with the timezones, that's the job of the Evaluator

**Reply by ghost** [reply]
There are some problems with time changes, because some events (f.e. advent of code) start in winter not at 0am, so in summer the time will be offset by one hour for this event.

(the testcase failed because of that)

**Comment by Ari24-cb24** [comment]
It's not the job of the parser to validate the user input. Even if there are some validations, the main part of the validation goes into the Evaluator.

**Comment by Ari24-cb24** [comment]
<blockquote>2 days after august</blockquote>

It's misleading but this actually means the third of august since august starts at the first. Another argument would be 48 hours added to the start (1st) of august.

**Comment by Ari24-cb24** [comment]
When we talked about DateTimeDelta Constants, we decieded against location tracking. Example:
  
<blockquote>noon, sunset, sunrise, dusk, dawn</blockquote>
We could've used the library [astral](https://pypi.org/project/astral) but we didn't want any locations for the user input. A strong argument for this was that we don't know where the user is (e.g. when being used in a bot). We decieded for some constants
